FirebaseAppDistributionInternal/Sources/AppDistributionApiService.swift

```diff
@@ -23,6 +23,13 @@ public typealias AppDistributionFetchReleasesCompletion = (_ releases: [Any]?, _
   -> Void
 public typealias AppDistributionFindReleaseCompletion = (_ releaseName: String?, _ error: Error?)
   -> Void
+public typealias AppDistributionCreateFeedbackCompletion = (_ feedbackName: String?,
```

Comment:
What's the reason for typealias's? We've usually found that, unlike typedefs in Objective-C, typealias's make code less readable in Swift, especially for callbacks where Xcode's code completion works less well.

Reply:
No resason-- this was just a relic of porting the existing code over from Objective-C. Inlined these and removed the typaliases.